src/components/QuranReader/MulkLearningPlanBanner/index.tsx

```
@@ -0,0 +1,122 @@
+          </Link>
+
+          <Button
+            size={ButtonSize.Medium}
+            href={LEARNING_PLAN_URL}
+            className={styles.mobileButton}
+            ariaLabel={t('mulk-lp.aria-cta')}
+          >
+            {t('mulk-lp.cta-text')}
+          </Button>
+
+          <div className={styles.desktopCaptionRow} aria-labelledby="mulk-banner-caption">
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-labelledby' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)